app/v1/controllers/AmendOtherDeductionsController.scala

```diff
@@ -42,14 +43,21 @@ class AmendOtherDeductionsController @Inject()(val authService: EnrolmentsAuthSe
                                                service: AmendOtherDeductionsService,
                                                auditService: AuditService,
                                                hateoasFactory: HateoasFactory,
-                                               cc: ControllerComponents)(implicit ec: ExecutionContext)
+                                               cc: ControllerComponents,
+                                               idGenerator: IdGenerator)(implicit ec: ExecutionContext)
   extends AuthorisedController(cc) with BaseController with Logging {
 
   implicit val endpointLogContext: EndpointLogContext =
     EndpointLogContext(controllerName = "AmendOtherDeductionsController", endpointName = "amendOtherDeductions")
 
   def handleRequest(nino: String, taxYear: String): Action[JsValue] =
     authorisedAction(nino).async(parse.json) { implicit request =>
+
+      implicit val correlationId: String = idGenerator.generateCorrelationId
+      logger.info(
+        s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
+          s"with CorrelationId: $correlationId")
```

Comment:
This change is unrelated to the sub-task for this branch and should have probably been included in a separate PR.